### ISSUE TITLE
[VoiceOver] Collection cells should be an accessibility element

### ISSUE THREAD

**Issue by romoore** [material-foundation/cocoapods-catalog-by-convention#28]
The CBC cells (main list/tiles) are not accessibility elements. VoiceOver users can only tap on the labels within each cell. Instead, the entire cell should be the accessibility element and should provide the values/labels/traits of the label and act as a button.
